src/ontology/cl-edit.owl

```diff
@@ -27043,6 +27047,38 @@ EquivalentClasses(obo:CL_3000004 ObjectIntersectionOf(obo:CL_0000111 ObjectSomeV
 SubClassOf(obo:CL_3000004 obo:CL_0000101)
 SubClassOf(obo:CL_3000004 obo:CL_0000111)
 
+# Class: obo:CL_4023003 (nuclear chain fiber)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1016/B978-0-323-39632-5.00024-4") Annotation(oboInOwl:hasDbXref "PMID:139469") obo:IAO_0000115 obo:CL_4023003 "A type of intrafusal muscle fiber that have nuclei arranged in a linear row, and the equatorial region is not obviously expanded. are responsible for the detection of changes in muscle length. They are innervated by static gamma motor neurons and are principally associated with type II sensory fiber")
+AnnotationAssertion(oboInOwl:created_by obo:CL_4023003 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(rdfs:label obo:CL_4023003 "nuclear chain fiber")
+SubClassOf(obo:CL_4023003 obo:CL_0008047)
+
+# Class: obo:CL_4023004 (nuclear bag fiber)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1016/B978-0-323-39632-5.00024-4") Annotation(oboInOwl:hasDbXref "PMID:139469") obo:IAO_0000115 obo:CL_4023004 "A type of intrafusal muscle fiber that lies in the center of a muscle spindle. Nuclei are clustered centrally and give the equatorial region a swollen appearance. They are associated with associated with dynamic gamma motor neurons, and the stretching of the equatorial region of the nuclear bag fibers results in an increase in the firing rate of type Ia sensory fibers.")
+AnnotationAssertion(oboInOwl:created_by obo:CL_4023004 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(rdfs:label obo:CL_4023004 "nuclear bag fiber")
+SubClassOf(obo:CL_4023004 obo:CL_0008047)
+
+# Class: obo:CL_4023005 (dynamic nuclear bag fiber)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1016/B978-0-323-39632-5.00024-4") obo:IAO_0000115 obo:CL_4023005 "They are sensitive mainly to the rate of change in muscle length.")
+AnnotationAssertion(oboInOwl:created_by obo:CL_4023005 <http://orcid.org/0000-0001-7258-9596>)
```

Comment:
A nuclear bag fiber that is sensitive mainly to the rate of change in muscle length.